[Breakpad] Missing file causing find_package(unofficial-breakpad) failure

**Describe the bug**
Since <https://github.com/microsoft/vcpkg/commit/146441473c8176a5509c6eac979860f37f834730>, it's impossible to configure a project with `find_package(unofficial-breakpad)`. CMake reports an error and can't complete configuration.

**Environment**
- OS: Confirmed on Windows and MacOS (should happen for others too)
- Compiler: N/A (it's a CMake issue)

**To Reproduce**
Put `find_package(unofficial-breakpad)` in your CMakeLists.txt.

**Expected behavior**
Proper detection of the library, as it was before the change.

**Failure logs**
Windows
```
1> [CMake] CMake Error at C:/vcpkg/installed/x64-windows-static/share/unofficial-breakpad/unofficial-breakpadConfig.cmake:6 (include):
1> [CMake]   include could not find requested file:
1> [CMake] 
1> [CMake]     C:/vcpkg/installed/x64-windows-static/share/unofficial-breakpad/unofficial-breakpad-targets.cmake
1> [CMake] Call Stack (most recent call first):
1> [CMake]   C:/vcpkg/scripts/buildsystems/vcpkg.cmake:859 (_find_package)
1> [CMake]   CMakeLists.txt:138 (find_package)
```

MacOS
```
  CMake Error at /usr/local/share/vcpkg/installed/x64-osx/share/unofficial-breakpad/unofficial-breakpadConfig.cmake:6 (include):
    include could not find requested file:
  
      /usr/local/share/vcpkg/installed/x64-osx/share/unofficial-breakpad/unofficial-breakpad-targets.cmake
  Call Stack (most recent call first):
    /usr/local/share/vcpkg/scripts/buildsystems/vcpkg.cmake:859 (_find_package)
    CMakeLists.txt:138 (find_package)
```

--- a/ports/breakpad/unofficial-breakpadConfig.cmake
+++ b/ports/breakpad/unofficial-breakpadConfig.cmake
@@ -3,4 +3,4 @@ if(@USED_ZLIB@)
   find_dependency(ZLIB)
 endif()
 
-include("${CMAKE_CURRENT_LIST_DIR}/unofficial-breakpad-targets.cmake")
+include("${CMAKE_CURRENT_LIST_DIR}/unofficial-breakpadTargets.cmake")

--- a/ports/breakpad/vcpkg.json
+++ b/ports/breakpad/vcpkg.json
@@ -1,6 +1,7 @@
 {
   "name": "breakpad",
   "version-date": "2023-01-27",
+  "port-version": 1,
   "description": "a set of client and server components which implement a crash-reporting system.",
   "homepage": "https://github.com/google/breakpad",
   "license": "BSD-3-Clause",

--- a/versions/b-/breakpad.json
+++ b/versions/b-/breakpad.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "5643527026fecb43d487919191b02c41bfced6f9",
+      "version-date": "2023-01-27",
+      "port-version": 1
+    },
+    {
       "git-tree": "b2b2523acc1c56222313e8fe01065c1e130440e1",
       "version-date": "2023-01-27",
       "port-version": 0

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -1314,7 +1314,7 @@
     },
     "breakpad": {
       "baseline": "2023-01-27",
-      "port-version": 0
+      "port-version": 1
     },
     "brigand": {
       "baseline": "1.3.0",

